### PR TITLE
added the Slack Community invite url to the PageFooter and the StayInformed page

### PR DESCRIPTION
**Pull request by danielkroon**

### CODE REVIEW

**Comment by fhoeben** [comment]
You can probably also remove the references to the Yahoo group. See also https://github.com/unclebob/fitnesse/issues/1250

**Comment by danielkroon** [comment]
> You can probably also remove the references to the Yahoo group. See also [unclebob/fitnesse#1250](https://github.com/unclebob/fitnesse/issues/1250)

@fhoeben sure! I wasn't sure if this reference could be removed completely.

File: FitNesseRoot/PlugIns/FitServers/DelphiFit/content.txt [hunk 1]
```diff
@@ -6,9 +6,6 @@ Updated to FIT20070619 and extended by Michal Wojcik: mgwojcik@gmail.com
 !3 Downloads:
 Project moved to http://code.google.com/p/fit4delphi/
 
-!3 News group:
```
Comment (fhoeben) [inline FitNesseRoot/PlugIns/FitServers/DelphiFit/content.txt]: I believe we should leave this page as-is.

**Review by fhoeben — CHANGES_REQUESTED** [review]

File: FitNesseRoot/PlugIns/FitServers/DelphiFit/content.txt (hunk 1 above)
Comment (danielkroon) [reply FitNesseRoot/PlugIns/FitServers/DelphiFit/content.txt]: Alright, I will undo the changes.

**Pull request merged**